bindings/python/client.py

```diff
@@ -306,13 +303,7 @@ def main():
         write_line(console, '(q)uit), (p)ause), (u)npause), (r)eannounce\n')
         write_line(console, 76 * '-' + '\n')
 
-        while 1:
-            a = ses.pop_alert()
-            if not a: break
-            alerts.append(a)
-
-        if len(alerts) > 8:
-            del alerts[:len(alerts) - 8]
+        alerts = ses.pop_alerts()
```

Comment:
this is a great fix btw. I'm surprised I've left that old code around for so long. I don't even get why it's deleting alerts beyond index 8.